modules/client-configuration/pages/supported-features-raspberrypios.adoc

```diff
@@ -0,0 +1,162 @@
+[[supported-features-raspberrypios]]
+= Supported {raspberrypios} Features
+
+This table lists the availability of various features on {raspberrypios} clients.
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{raspberrypios} is supported by the {raspberrypios} community.
+====
+
+The icons in this table indicate:
+
+* {check} the feature is available on both Salt and traditional clients
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* {trad} the feature is supported only on traditional clients
+* {salt} the feature is supported only on Salt clients.
+
+
+[cols="1,1,1,1", options="header"]
```

Comment:
This is not rendering correctly:

![image](https://github.com/uyuni-project/uyuni-docs/assets/4226070/a32268a2-7878-4d3e-9c0b-6329e7ca00f3)

I am not sure, but I think you need to adjust `cols` to `1,1`